src/dmd/dtemplate.d

```diff
@@ -5984,6 +5984,10 @@ extern (C++) class TemplateInstance : ScopeDsymbol
         for (TemplateInstance cur = this; cur; cur = cur.tinst)
         {
             ++n_instantiations;
+            // Set error here as we don't want it to depend on the number of
+            // entries that are being printed.
+            cur.errors = true;
```

Comment:
What caused this commit?

Reply:
It just seems very wrong to have this set *only* on the templates we show (which can vary depending on `-v`).
`errors` has meaning semantic-wise, and the amount of frames we print should not affect the semantic of the code.
I didn't witness anything wrong that prompted the commit, it's just my observation of what *should* happen.
Note that I believe the diff on `test/fail_compilation/deprecations.d` (L9 in the old code) is due to this: We don't show the error message twice anymore.